mods/default/furnace.lua

```diff
@@ -237,6 +243,12 @@ local function furnace_node_timer(pos, elapsed)
 		swap_node(pos, "default:furnace_active")
 		-- make sure timer restarts automatically
 		result = true
+
+		-- Play sound every 5 seconds while the furnace is active
+		if timer_elapsed == 0 or (timer_elapsed+1) % 5 == 0 then
```

Comment:
since the sound is 10 seconds long why not play it e.g. every 9 seconds?

Reply:
I cannot remember the exact reason why it was set to 5 seconds, but we discussed it here:
https://github.com/minetest/minetest_game/pull/2569#issuecomment-584960027

Reply:
Probably because of the fade in/out durations.

Reply:
it's fine then.